lib/config.js

```diff
@@ -10,7 +10,12 @@ var truncate = require('unicode-byte-truncate')
 var entries = require('object.entries') // TODO: Remove when Node.js 6 is no longer supported
 
 var version = require('../package').version
-var userAgent = 'elastic-apm-node/' + version
+var packageName = require('../package').name
+// Standardize user-agent header. Only use "elasticapm-node" if it matches "elastic-apm-node". 
+if (packageName === 'elastic-apm-node') {
+  packageName = 'elasticapm-node'
+}
+var userAgent = `${packageName}/${version}`
```

Comment:
@axw Do you think we should be doing this package name check or should we just _always_ call it `elasticapm-node`? I'm not sure there's a significant value to being able to differentiate forks. 🤔

Reply:
I'm not personally fussed either way. Could be useful for debugging? OTOH, we can probably tell from stack traces anyway right? Are there any downsides?

Reply:
Not really, just extra code. 🤷‍♂

Reply:
Let's leave it in, could be useful for diagnosing issues in the future.